### PR TITLE
Auth filters and 1.5

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: _includes/manuals/1.5/4.1.2_roles_and_permissions.md [hunk 1]
```diff
@@ -1,22 +1,53 @@
-A user's access to the features of Foreman are constrained by the roles and permissions that they are granted. These permissions are also used to restrict the set of hosts, host groups and domains that a user is able to access and modify.
+A user's access to the features of Foreman are constrained by the **permissions** that they are granted. These permissions are also used to restrict the set of hosts, host groups and other resources that a user is able to access and modify.
 
 Note: a user with global admin enabled is not restricted by the authorization system. This is the default for installations that do not have :login:true in _config/settings.yml_.
 
-A logged in user will be granted the **Anonymous** role plus one or more additional roles. The permissions associated with these roles are aggregated and determine the final permission set.
+A logged in user will be granted the **Anonymous** role plus one or more additional roles. The permissions and filters associated with these roles are aggregated and determine the final permission set.
 
 Roles may be administered only by a user with global admin privileges.
```
Comment (domcleal) [inline _includes/manuals/1.5/4.1.2_roles_and_permissions.md]: I think the permissions to manage roles can now be delegated too, so this is no longer true?

File: _includes/manuals/1.5/4.1.2_roles_and_permissions.md [hunk 2]
```diff
@@ -1,22 +1,53 @@
...
 
 #### Roles
 
-These may be created, deleted and edited on the **Roles** page. Each role will be associates with one or more base privileges.
+These may be created, deleted and edited on the **Roles** page. Each role will contain permission filters, which define the actions allowed in a certain resource. Once your role is created, you can associate it with one or more users and user groups.
 
 There are two builtin system roles
 1. Anonymous: This is a set of permissions that every user at your installation will be granted, irrespective of any other roles that they have.
```
Comment (domcleal) [inline _includes/manuals/1.5/4.1.2_roles_and_permissions.md:16]: Needs a linebreak above, formatting's a bit broken.

File: _includes/manuals/1.5/4.1.2_roles_and_permissions.md [hunk 3]
```diff
@@ -1,22 +1,53 @@
...
+
+3. Fact filtering: 'facts.alarmlevel = high' - Will apply permissions to hosts with a fact 'alarmlevel' with value 'high'. As a fact is only generated during a puppet run, this filter will only refer to machines that have been built and therefore cannot be used to restrict the creation of machines.
+
+These pools of queries can be combined by adding them together or the filters can be used to restrict the selected resource to a smaller and smaller subset of the total. Think of them as set operations.
+
+Note: If the "Administrator" check box is checked for a user, filtering will not take effect.
+
+#### Example
+
+A user's filter section has the **select** checkbox unticked, indicating that the user's owned hosts are not included in the final set of hosts, (unless they are selected some other way.)
```
Comment (domcleal) [inline _includes/manuals/1.5/4.1.2_roles_and_permissions.md]: None of these filters exist any more.. I think this section needs updating?

Comment (dLobatog) [inline _includes/manuals/1.5/4.1.2_roles_and_permissions.md]: Sorry, I pushed an old version of this, will resubmit in a second.

**Comment by domcleal** [comment]
Merged, thanks!

**Pull request merged**